backend/src/main/java/org/sefglobal/core/repository/AmbassadorRepository.java

```diff
@@ -10,4 +12,5 @@
 public interface AmbassadorRepository extends JpaRepository<Ambassador, Long> {
```

Comment:
Use `PagingAndSortingRepository` instead

Reply:
Isn't the `JpaRepository` extends the `PagingAndSortingRepository `?

Reply:
According to @anjula-sack 's point, I think `JpaRepository` is fine.